docs/docsite/rst/community/maintainers_guidelines.rst

```diff
@@ -39,32 +39,31 @@ To nominate a candidate, create a GitHub issue in the relevant collection reposi
 Communicating as a collection maintainer
 -----------------------------------------
 
- Maintainers MUST subscribe to the `"Changes impacting collection contributors and maintainers" GitHub repo <https://github.com/ansible-collections/news-for-maintainers>`_ and the `Bullhorn newsletter <https://github.com/ansible/community/wiki/News#the-bullhorn>`_. If you have something important to announce through the newsletter (for example, recent releases), see the `Bullhorn's wiki page <https://github.com/ansible/community/wiki/News#the-bullhorn>`_ to learn how.
+ Maintainers are highly encouraged to subscribe to the `"Changes impacting collection contributors and maintainers" GitHub repo <https://github.com/ansible-collections/news-for-maintainers>`_ and the `Bullhorn newsletter <https://github.com/ansible/community/wiki/News#the-bullhorn>`_. If you have something important to announce through the newsletter (for example, recent releases), see the `Bullhorn's wiki page <https://github.com/ansible/community/wiki/News#the-bullhorn>`_ to learn how.
 
 
-Collection contributors and maintainers should also communicate through:
+Collection contributors and maintainers can also communicate through:
 
-* :ref:`communication_irc` appropriate to their collection, or if none exists, the general community and developer chat channels
-* Mailing lists such as `ansible-announce <https://groups.google.com/d/forum/ansible-announce>`_ and `ansible-devel <https://groups.google.com/d/forum/ansible-devel>`_
-* Collection project boards, issues, and GitHub discussions in corresponding repositories
-* Quarterly Contributor Summits.
+* Real-time chats and forum topics appropriate to their collection, or if none exists, the general community and developer chat channels.
+* Collection project boards, issues, and GitHub discussions in corresponding repositories.
+* Contributor Summits and Ansible community days.
 * Ansiblefest and local meetups.
 
 See :ref:`communication` for more details on these communication channels.
 
 .. _wg_and_real_time_chat:
 
 Establishing working group communication
-----------------------------------------------------------------
+----------------------------------------
 
-Working groups depend on efficient, real-time communication.
+Working groups depend on efficient forum and/or real-time communication.
```

Comment:
```suggestion
Working groups depend on an efficient forum and good communication.
```

Reply:
I think your changes change the meaning a little too much:)
How about `Working groups depend on an efficient communication.` ?
I think no need to mention the forum or online chat if we have it explained below anyway.
If you're OK, please change your comment, then we commit it.